### PR TITLE
Misc. cleanup of the program/lwaftr/ subtree

### PR DESCRIPTION
**Pull request by teknico**
It includes one bugfix.

### CODE REVIEW

File: src/program/lwaftr/run_nohw/run_nohw.lua [hunk 1]
```diff
@@ -75,7 +73,7 @@ function run(parameters)
    config.link(c, "aftr.v6 -> b4if.rx")
 
    if verbosity >= 1 then
-      local csv = CSVStatsTimer.new(csv_file)
+      local csv = CSVStatsTimer.new(bench_file)
```
Comment (teknico) [inline src/program/lwaftr/run_nohw/run_nohw.lua:76]: This one is an actual bugfix.

Comment (kbara) [reply src/program/lwaftr/run_nohw/run_nohw.lua:76]: Thanks for commenting that.

**Comment by kbara** [comment]
Why did the tests fail?

**Comment by teknico** [comment]
The soaktest failed. It doesn't seem related.

**Comment by dpino** [comment]
@teknico I think it's in fact related. The patch localizes `load_soaktest` and now it cannot be found in `lwaftr/soaktest/soaktest.lua`. 

https://github.com/Igalia/snabb/pull/501/files#diff-2250890b41fc25cc75a5f715d0ef08b8R309

When CI fails, check what's the error and run `make test` locally in your branch. If you get the same error as CI reports, there's something with the PR. If your PR passes locally but CI fails, then there's something going wrong with snabb-bot.

**Comment by teknico** [comment]
Hopefully fixed the mess I made, sorry about that. Added some more cleanup too.

**Comment by kbara** [comment]
```
src/testlog/program.lwaftr.tests.soaktest.selftest.sh:
Testing: Regression test: make sure ntohl'd high-bit-set fragment IDs do not crash
loading source binding table from ../data/binding-table.txt
done
program/lwaftr/soaktest/soaktest.lua:39: variable 'load_soak_test_on_a_stick' is not declared
stack traceback:
    core/main.lua:137: in function <core/main.lua:135>
    [C]: in function 'error'
    lib/lua/strict.lua:37: in function '__index'
    program/lwaftr/soaktest/soaktest.lua:39: in function 'run'
    program/lwaftr/lwaftr.lua:17: in function 'run'
    core/main.lua:56: in function <core/main.lua:43>
    [C]: in function 'xpcall'
    core/main.lua:179: in main chunk
    [C]: at 0x004523e0
    [C]: in function 'pcall'
    core/startup.lua:3: in main chunk
    [C]: in function 'require'
[string "require "core.startup""]:1: in main chunk
```

Might be worth running 'make test' locally, or at least the lwaftr-related selftest.sh scripts.

**Comment by teknico** [comment]
Oh dear. I definitely did run the tests locally, and I could swear I had "unlocalized" that too. :-(

**Comment by kbara** [comment]
Happens to the best of us. :D

**Comment by kbara** [comment]
LGTM

**Pull request merged**